lib/govspeak.rb

```diff
@@ -141,9 +141,14 @@ def legislative_list_footnote_definitions(source)
         @footnote_definition_html = "<div class=\"footnotes\" role=\"doc-endnotes\">\n<ol>"
 
         while counter < footnotes.size
+          footnote_definition = Govspeak::Document.new(footnotes[counter][1].strip).to_html[/(?<=<p>).*(?=<\/p>)/]
```

Comment:
Nice workaround 👍 my only concern is the regex inside the `to_html` here. I guess this is to add `<p>` tags to the footnotes, which replicates how the 'normal' footnotes work, but if ever there is a change to the HTML that footnotes should be nested within, we'd have to remember to update this one too.

Is there a common presenter somewhere that we can delegate this logic to rather than having it inline here?

Reply:
Ah so this was something handy I discovered which uses using positive lookbehind and positive lookahead in the regex - `/(?<=<p>).*(?=<\/p>)/` finds all the characters between a `<p>` and `</p>` tag, but does not include the tags.
From what I understand they will always be wrapped in `<p>` tags if the spacing is correct in the markdown.

Reply:
Ah ok - so the `Govspeak::Document.new` takes the footnote markdown, we call `.to_html` on it to convert it from markdown to HTML, but we use the regex bit to strip out the `<p>` tags because they'll be added in automatically later on? Is that right?

Reply:
Yep that's right!